### PR TITLE
refactor(ivy): drop the `element` prefix from all styling instructions

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

**Comment by matsko** [comment]
https://tap.corp.google.com/ui#id=OCL:247264863:BASE:247264901:1557343270884:57462fb0

File: packages/compiler/src/render3/view/styling_builder.ts [hunk 1]
```diff
@@ -277,18 +277,18 @@ export class StylingBuilder {
...
-   * Builds an instruction with all the expressions and parameters for `elementStyling`.
+   * Builds an instruction with all the expressions and parameters for `styling`.
    *
    * The instruction generation code below is used for producing the AOT statement code which is
    * responsible for registering style/class bindings to an element.
    */
-  buildElementStylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool):
-      Instruction|null {
-    const reference = this._directiveExpr ? R3.elementHostStyling : R3.elementStyling;
+  buildstylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool): Instruction
```
Comment (kara) [inline packages/compiler/src/render3/view/styling_builder.ts]: Seems like we lost the camel-case in the find-and-replace.

```suggestion
  buildStylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool): Instruction
```

File: packages/compiler/src/render3/view/styling_builder.ts [hunk 2]
```diff
@@ -340,25 +335,25 @@ export class StylingBuilder {
...
   /**
-   * Builds an instruction with all the expressions and parameters for `elementClassMap`.
+   * Builds an instruction with all the expressions and parameters for `classMap`.
    *
-   * The instruction data will contain all expressions for `elementClassMap` to function
+   * The instruction data will contain all expressions for `classMap` to function
    * which includes the `[class]` expression params.
    */
-  buildElementClassMapInstruction(valueConverter: ValueConverter): Instruction|null {
+  buildclassMapInstruction(valueConverter: ValueConverter): Instruction|null {
```
Comment (kara) [inline packages/compiler/src/render3/view/styling_builder.ts]: Same for these

```suggestion
  buildClassMapInstruction(valueConverter: ValueConverter): Instruction|null {
```

File: packages/compiler/src/render3/view/styling_builder.ts [hunk 3]
```diff
@@ -340,25 +335,25 @@ export class StylingBuilder {
...
   /**
-   * Builds an instruction with all the expressions and parameters for `elementStyleMap`.
+   * Builds an instruction with all the expressions and parameters for `styleMap`.
    *
-   * The instruction data will contain all expressions for `elementStyleMap` to function
+   * The instruction data will contain all expressions for `styleMap` to function
    * which includes the `[style]` expression params.
    */
-  buildElementStyleMapInstruction(valueConverter: ValueConverter): Instruction|null {
+  buildstyleMapInstruction(valueConverter: ValueConverter): Instruction|null {
```
Comment (kara) [inline packages/compiler/src/render3/view/styling_builder.ts]:
```suggestion
  buildStyleMapInstruction(valueConverter: ValueConverter): Instruction|null {
```

**Review by kara — APPROVED** [review]
LGTM, once camel case is fixed

**Review by alxhub — APPROVED** [review]

**Pull request closed without merging**